install.sh

```diff
@@ -8,28 +8,65 @@
-  echo "    [-k|--relwithdebinfo] -DCMAKE_BUILD_TYPE=RelWithDebInfo"
-  echo "    [-r]--relocatable] create a package to support relocatable ROCm"
-  echo "    [--cuda|--use-cuda] build library for cuda backend"
-  echo "    [--[no-]hip-clang] Whether to build library with hip-clang"
-  echo "    [--compiler] specify host compiler"
-  echo "    [-p|--cmakepp] addition to CMAKE_PREFIX_PATH"
-  echo "    [--custom-target] link against custom target (e.g. host, device)"
-  echo "    [-v|--rocm-dev] Set specific rocm-dev version"
-  echo "    [-b|--rocblas] Set specific rocblas version"
-  echo "    [--rocblas-path] Set specific path to custom built rocblas"
-  echo "    [--rocsolver-path] Set specific path to custom built rocsolver"
-  echo "    [--static] Create static library instead of shared library"
-  echo "    [--codecoverage] build with code coverage profiling enabled"
-  echo "    [--address-sanitizer] Build with address sanitizer enabled. Uses hipcc as compiler"
+cat <<EOF
+
+  hipBLAS library build & installation helper script.
+
+  Usage:
+    $0 (build hipblas and put library files at ./build/hipblas-install)
+    $0 <options> (modify default behavior according to the following flags)
+
+  Options:
+    -h | --help                   Print this help message.
+
+    -i | --install                Pass this flag to generate and install library package after build.
+
+    -d | --dependencies           Pass this flag to also build and install external dependencies.
+                                  Dependecies are to be installed in /usr/local. This should be done only once.
+                                  (this does not install rocBLAS or rocSolver)
+
+    -c | --clients                Pass this flag to also build the library clients benchmark and gtest.
+                                  (Generated binaries will be located at builddir/clients/staging)
+
+    -n | --no-solver              Pass this flag to build hipLBAS library without rocSOLVER dependency
+
+    -g | --debug                  Pass this flag to build in Debug mode (equivalent to set CMAKE_BUILD_TYPE=Debug).
+                                  (Default build type is Release)
+
+    -s | --static                 Pass this flag to build hipblas as a static library.
+                                  (hipblas must be built statically when the used companion rocblas is also static).
+
+    -r | --relocatable            Pass this to add RUNPATH(based on ROCM_RPATH) and remove ldconf entry.
+
+    -k | --relwithdebinfo         Pass this flag to build in release debug mode (equivalent to set CMAKE_BUILD_TYPE=RelWithDebInfo).
+                                  (Default build type is Release)
+
+    --rocblas-path <blasdir>      Specify path to an existing rocBLAS install directory.
+                                  (e.g. /src/rocBLAS/build/release/rocblas-install)
+
+    -b | --rocblas                Pass this flag to specify rocblas version.
+
```

Comment:
should this be 
-b | --rocblas <version>

Reply:
Yes, I will fix it